peakina/readers/excel.py

```diff
@@ -0,0 +1,133 @@
+"""
+Module to add xml support
+"""
+import enum
+from io import StringIO
+from typing import Any, List, Optional
+
+import pandas as pd
+import xlrd
+from openpyxl import load_workbook
+from openpyxl.utils.exceptions import InvalidFileException
+
+from peakina.readers.csv import PreviewArgs
```

Comment:
Probably should be somewhere else as it's common for all "preview", which is a custom kwarg that we add. Maybe put it in `helpers`?